src/data/content/community/media/20230518-accessibility-days-2023.yaml

```
@@ -0,0 +1,114 @@
+# ————————————————————————————————————————————
+# CREATEPAGE GATSBY
+# ————————————————————————————————————————————
+metadata:
+  template:
+    name: level3 # article?
+  archive: media
+
+seo:
+  name: "Il design system del Paese e la responsabilità di progettare interfacce pubbliche - Designers Italia"
+  description: Daniele Tabellini e Fabrizio Caccavelo per gli Accessibility Days 2023, Roma
+  image: /images/community/media-thumbnails/20230518-accessibility-days-2023.jpg
+  twitterImage: null
+  canonical: null
+  pathname: /community/media/20230518-accessibility-days-2023
+
+kangaroo:
+  navposition:
+    id: kangEnd
+    items:
+      - title: Ti trovi in
+        icon:
+          icon: "sprites.svg#it-folder"
+          size: "sm"
+          align: "middle"
+          color: "secondary"
+        label: Media
+        url: "../"
+
+components:
+  hero:
+    background: null
+    centered: true
+    title: 'Il design system del Paese e la responsabilità di progettare interfacce pubbliche'
+    subtitle: Daniele Tabellini e Fabrizio Caccavelo per gli Accessibility Days 2023, Roma
+    tag:
+      label: Intervento
+      addonClasses: mt-3 text-uppercase
+
+    kangaroo:
+      id: kangarooExample
+      titleSr: Metadati e link per approfondire
+      noPadding: true
+      tagsLabel: Parliamo di
+      color: secondary
+      icon:
+        icon: sprites.svg#it-horn
+        size: sm
+        color: secondary
+      tags: [Esperienza utente,Accessibilità]
+
+  sectionsEditorial:
+    # FORMATO VIDEO FULL
+    # - menu: false
+    #   fullColumn: true
+    #   full: true
+    #   noSpace: true
+    #   centered: true
+    #   components:
+    #     - name: MediaPlayer
+    #       lang: it
+    #       url: https://youtu.be/_0j7ZQ67KtY
+    #       poster: /images/community/media-thumbnails/2023-08-21-media-thumbnail.jpg
+    #       trascription: Vestibulum hendrerit ultrices nibh
+
+    # FORMATO VIDEO DEFAULT
+    - fullColumn: false # fullwidth vs normal
+      full: true # large column vs normal
+      noSpace: false 
+      centered: true
+      background: medium
+      components:
+        - name: MediaPlayer
+          lang: it
+          url: https://youtu.be/py9RgGDrOtM?t=25
+          poster: /images/community/media-thumbnails/20230518-accessibility-days-2023.jpg
+          trascription: |
+            Vai alla [scheda del video su YouTube](https://youtu.be/py9RgGDrOtM?t=25){:target="_blank"}
```

```suggestion
            Vai alla [scheda del video su YouTube](https://youtu.be/py9RgGDrOtM?t=25)
```

We need to remove this @bfabio @astagi. It's an a11y error without telling the user you will open a new window, we need at least an aria-label for that. But I would prefer to manage this kind of stuff on markdown links centrally, see issue https://github.com/italia/designers.italia.it/issues/1005